### PR TITLE
Admin Panel Updates from Shanine

### PR DESCRIPTION
**Pull request by sdespacio**
## Because

Resolving issues in regards to the Admin Panel.

- Renamed all instances of `EmailBlocks` or `email-blocks` to `AccountSearch` or `account-search`.
- Added new menu buttons on the navigation bar: **Admin Logs** (related to issue #6757), and **Site Status** (related to issue #6756)
- Added new svg files in the `Images` folder for the menu buttons
- Put a short sentence (temporarily) for the Admin Logs page

## This pull request

Renames all instances of `EmailBlocks` or `email-blocks` to `AccountSearch` or `account-search` on the admin panel.
Adds new menu buttons on the navigation bar: **Admin Logs** (related to issue #6757), and **Site Status** (related to issue #6756)

## Issue that this pull request solves

Closes: #6746 
Related to issues: #6757, #6756 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [ ] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)

image

### CODE REVIEW

File: packages/fxa-admin-panel/src/components/Nav/index.tsx [hunk 1]
```diff
@@ -12,13 +12,13 @@ export const Nav = () => (
...
         <li>
-          <NavLink exact to="/email-blocks">
+          <NavLink exact to="/account-search">
             <img
               className="inline-flex icon"
               src={require('../../images/icon-mail.svg')}
               alt="external link"
             />
-            Email blocks
+            Account search
```
Comment (akashmathur0) [inline packages/fxa-admin-panel/src/components/Nav/index.tsx:21]: Would we be able to capitalize the both words to 'Account Search'?

**Review by 6a68 — DISMISSED** [review]
Thanks for the pull request @sdespacio! This looks good so far.

If the longer-term plan is to make this an account search component, it probably would make sense to rename the component itself from EmailBlocks to AccountSearch. You can find occurrences of the string "EmailBlocks" using a search tool, like `ripgrep`:

```sh
 ]$ rg EmailBlocks
App.tsx
8:import EmailBlocks from './components/EmailBlocks';
15:        <Route path="/account-search" component={EmailBlocks} />

components/EmailBlocks/index.tsx
52:export const EmailBlocks = () => {
142:export default EmailBlocks;

components/EmailBlocks/index.test.tsx
11:import { GET_ACCOUNT_BY_EMAIL, EmailBlocks } from './index';
110:  const renderResult = render(<EmailBlocks />);
130:      <EmailBlocks />
171:      <EmailBlocks />
```

Go ahead and rename the `src/components/EmailBlocks` directory to `src/components/AccountSearch` (use `git mv`) and update the occurrences of `EmailBlocks` to `AccountSearch` throughout (use rg or grep to help you find all the spots, like I did above). 

Please feel free to ask for help here or on Slack if you have any questions, or if you get stuck for more than a few minutes.

**Comment by sdespacio** [comment]
> Thanks for the pull request @sdespacio! This looks good so far.
> 
> If the longer-term plan is to make this an account search component, it probably would make sense to rename the component itself from EmailBlocks to AccountSearch. You can find occurrences of the string "EmailBlocks" using a search tool, like `ripgrep`:
> 
> ```shell
>  ]$ rg EmailBlocks
> App.tsx
> 8:import EmailBlocks from './components/EmailBlocks';
> 15:        <Route path="/account-search" component={EmailBlocks} />
> 
> components/EmailBlocks/index.tsx
> 52:export const EmailBlocks = () => {
> 142:export default EmailBlocks;
> 
> components/EmailBlocks/index.test.tsx
> 11:import { GET_ACCOUNT_BY_EMAIL, EmailBlocks } from './index';
> 110:  const renderResult = render(<EmailBlocks />);
> 130:      <EmailBlocks />
> 171:      <EmailBlocks />
> ```
> 
> Go ahead and rename the `src/components/EmailBlocks` directory to `src/components/AccountSearch` (use `git mv`) and update the occurrences of `EmailBlocks` to `AccountSearch` throughout (use rg or grep to help you find all the spots, like I did above).
> 
> Please feel free to ask for help here or on Slack if you have any questions, or if you get stuck for more than a few minutes.

Thank you @6a68! I'll implement these changes and re-submit :) !

File: packages/fxa-admin-panel/src/components/Nav/index.tsx (hunk 1 above)
Comment (sdespacio) [reply packages/fxa-admin-panel/src/components/Nav/index.tsx]: I've capitalized this with my latest commit :)!

File: packages/fxa-admin-panel/src/components/AdminLogs/index.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useState } from 'react';
+import { useLazyQuery, gql } from '@apollo/client';
```
Comment (dannycoates) [inline packages/fxa-admin-panel/src/components/AdminLogs/index.tsx:6]: Running `yarn workspace fxa-admin-panel build` produces these linter warnings.
```
./src/components/AdminLogs/index.tsx
  Line 5:17:  'useState' is defined but never used      @typescript-eslint/no-unused-vars
  Line 6:10:  'useLazyQuery' is defined but never used  @typescript-eslint/no-unused-vars
  Line 6:24:  'gql' is defined but never used           @typescript-eslint/no-unused-vars
```
We should delete the unused imports.

File: packages/fxa-admin-panel/src/components/Nav/index.tsx [hunk 3]
```diff
@@ -12,13 +12,33 @@ export const Nav = () => (
...
             />
-            Email blocks
+            Account Search
+          </NavLink>
+        </li>
+        <li>
+          <NavLink exact to="/site-status">
+            <img
+              className="inline-flex icon"
+              src={require('../../images/icon-site-status.svg')}
```
Comment (dannycoates) [inline packages/fxa-admin-panel/src/components/Nav/index.tsx:28]: I realize you're following the existing pattern from the mail icon here but I think we should prefer `import` over `require` when we can, so I'd like to change this to something like this:

```jsx
import './index.scss';
import mailIcon from '../../images/icon-mail.svg';
import statusIcon from '../../images/icon-site-status.svg';

...

<img src={statusIcon} ... />
```

**Review by dannycoates — COMMENTED** [review]
Looking good @sdespacio 

The build step, `yarn workspace fxa-admin-panel build` doesn't like to handle empty files. If we remove those it should build correctly.

File: packages/fxa-admin-panel/src/components/AdminLogs/index.tsx (hunk 2 above)
Comment (sdespacio) [reply packages/fxa-admin-panel/src/components/AdminLogs/index.tsx:6]: Thanks @dannycoates ! I'll add these fixes

**Comment by dannycoates** [comment]
This looks great @sdespacio. Last thing before we merge is to squash these 4 commits into one. The steps are pretty much the same as what I described [here](https://github.com/mozilla/fxa/pull/6818#issuecomment-726274663).

**Review by dannycoates — APPROVED** [review]

**Pull request merged**